activerecord/Rakefile

```diff
@@ -109,20 +127,38 @@ namespace :db do
     desc "Build the PostgreSQL test databases"
     task :build do
       config = ARTest.config["connections"]["postgresql"]
-      %x( createdb -E UTF8 -T template0 #{config["arunit"]["database"]} )
-      %x( createdb -E UTF8 -T template0 #{config["arunit2"]["database"]} )
+
+      if ENV["TRAVIS"]
+        %x(createdb -E UTF8 -T template0 #{config["arunit"]["database"]})
+        %x(createdb -E UTF8 -T template0 #{config["arunit2"]["database"]})
+      else
+        puts "\nCreating database '#{config['arunit']['database']}' as PostgreSQL user '#{config['arunit']['username']}'"
+        %x(createdb -E UTF8 -T template0 --host=#{config["arunit"]["host"]} --username=#{config["arunit"]["username"]} --password #{config["arunit"]["database"]})
+
+        puts "\nCreating database '#{config['arunit2']['database']}' as PostgreSQL user '#{config['arunit2']['username']}'"
+        %x(createdb -E UTF8 -T template0 --host=#{config["arunit2"]["host"]} --username=#{config["arunit2"]["username"]} --password #{config["arunit2"]["database"]})
```

Comment:
@vipulnsward, @rafaelfranca 

Here option `--password` used for  prompt for enter password!

Now example config looks like:

```yaml
  mysql2:
    arunit:
      username: rails
      encoding: utf8
      collation: utf8_unicode_ci
    arunit2:
      username: rails
      encoding: utf8

  postgresql:
    arunit:
      host: localhost
      username: rails
      min_messages: warning
    arunit2:
      host: localhost
      username: rails
      min_messages: warning
```
We can provide different credentials on machine as said in [docs](http://edgeguides.rubyonrails.org/development_dependencies_install.html#mysql-and-postgresql) if necessary.
Example: replace rows `username: rails` to `username: example_username`

After this pull request, creating, dropping databases looks like:
```bash
$ cd activerecord
$ bundle exec rake db:create

Creating database 'activerecord_unittest' as MySQL user 'example_username'
Enter password: 

Creating database 'activerecord_unittest2' as MySQL user 'example_username'
Enter password: 

Creating database 'activerecord_unittest' as PostgreSQL user 'example_username'
Password: 

Creating database 'activerecord_unittest2' as PostgreSQL user 'example_username'
Password: 
```
```bash
$ cd activerecord
$ bundle exec rake db:drop

Dropping database 'activerecord_unittest' as MySQL user 'example_username'
Enter password: 

Dropping database 'activerecord_unittest2' as MySQL user 'example_username'
Enter password: 

Dropping database 'activerecord_unittest' as PostgreSQL user 'example_username'
Password: 

Dropping database 'activerecord_unittest2' as PostgreSQL user 'example_username'
Password: 
```


@vipulnsward
> What's the purpose of username/password here? I believe this is assuming we are going to use the config for some "superuser" that has create privileges?

As i said "Connection should only works based on data that contains in activerecord/test/config.yml, instead of data from environment our OS or somewhere else."
It should work for all OS equally. Some people don't use superuser's name for PostgreSQL or MySQL.

LGTM. Your thoughts?